docs/pt/docs/tutorial/bigger-applications.md

```diff
@@ -0,0 +1,556 @@
+# Aplicações Maiores - Múltiplos Arquivos
+
+Se você está construindo uma aplicação ou uma API web, é raro que você possa colocar tudo em um único arquivo.
+
+**FastAPI** oferece uma ferramenta conveniente para estruturar sua aplicação, mantendo toda a flexibilidade.
+
+/// info
```

Comment:
I think it's nice to keep the translations of tip too:

`/// info | "Informação"`

Other translations are doing this:

![example](https://github.com/user-attachments/assets/3a39512b-808f-4bae-b61d-0cfaa081365a)